### PR TITLE
created orders controller

### PR DESCRIPTION
**Pull request by lynguillu**
+1

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Order < ApplicationRecord
+  belongs_to :customer, optional: true
+  has_many :carted_products
+  has_many :customers, through: :carted_products
+end
```
Comment (DillonStreator) [inline app/models/order.rb]: I believe that on line 4 you meant :products instead of :customers?

Comment (DillonStreator) [inline app/models/order.rb]: Does an order belong to a customer and also have many customers?

Comment (dillonstreator) [inline app/models/order.rb:5]: An order belongs to a customer and has many customers?

Comment (heydanhey) [reply app/models/order.rb:5]: This is just a work in progress. I don't think this is intended for merging.

**Pull request closed without merging**